contracts/contracts/othellofactory.sol

```diff
@@ -2,49 +2,85 @@ pragma solidity >=0.5.0 <0.6.0;
 pragma experimental ABIEncoderV2;
 
 import "./othelloboard.sol";
-
-contract othellofactory is othelloboard{
+import "./Board.sol";
+contract othellofactory is Board{

```

Comment:
Please use a HAS-A relationship rather than a IS-A relationship.

https://ethereum.stackexchange.com/questions/1415/solidity-create-contract-from-contract

Trying to inherit from Board means that we waste lots of effort copying state between struct Game and the Board contract.